### PR TITLE
Clarify Enterprise required for multiple instances

### PR DESCRIPTION
**Pull request by bbodenmiller**
Update docs to match main readme

Please squash merge

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ This plugin supports a two-way integration between Mattermost and Jira. For a st
 
 This plugin supports Jira Core and Jira Software products for Server, Data Center, and Cloud platforms. It has been tested with versions 7 and 8.
 
-**Support for multiple Jira instances is supported from Jira 3.0 and configured using [Administrator Slash Commands](https://mattermost.gitbook.io/plugin-jira/admininstrator-guide/administrator-slash-commands). Available for Mattermost Enterprise E20.**
+**Support for multiple Jira instances is supported from Jira 3.0 for Mattermost Enterprise Editions and configured using [Administrator Slash Commands](https://mattermost.gitbook.io/plugin-jira/admininstrator-guide/administrator-slash-commands).**
```
Comment (mickmister) [inline docs/README.md:9]: An E20 license is required to use the feature, i.e. an instance with an E10 enterprise license won't work. Can we keep the part about E20 in to be clear?

**Review by mickmister — CHANGES_REQUESTED** [review]

**Review by mickmister — APPROVED** [review]
Great work, thanks @bbodenmiller!

File: docs/README.md [hunk 2]
```diff
@@ -6,4 +6,4 @@ This plugin supports a two-way integration between Mattermost and Jira. For a st
 
 This plugin supports Jira Core and Jira Software products for Server, Data Center, and Cloud platforms. It has been tested with versions 7 and 8.
 
-**Support for multiple Jira instances is supported from Jira 3.0 and configured using [Administrator Slash Commands](https://mattermost.gitbook.io/plugin-jira/administrator-guide/administrator-slash-commands). Available for Mattermost Enterprise E20.**
+**Support for multiple Jira instances is supported from Jira 3.0 for Mattermost Enterprise E20 and configured using [Administrator Slash Commands](https://mattermost.gitbook.io/plugin-jira/administrator-guide/administrator-slash-commands).**
```
Comment (justinegeffen) [inline docs/README.md:9]:
```suggestion
**Support for multiple Jira instances is supported from Jira 3.0 for Mattermost Enterprise Edition E20 and configured using [Administrator Slash Commands](https://mattermost.gitbook.io/plugin-jira/administrator-guide/administrator-slash-commands).**
```

File: readme.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 This plugin supports a two-way integration between Mattermost and Jira. Jira Core and Jira Software products, for Server, Data Center, and Cloud platforms are supported. It has been tested with versions 7 and 8.
 
-From v3.0 of this plugin, support for multiple Jira instances is offered for Mattermost Enterprise Editions.
+From v3.0 of this plugin, support for multiple Jira instances is offered for Mattermost Enterprise E20.
```
Comment (justinegeffen) [inline readme.md:5]:
```suggestion
From v3.0 of this plugin, support for multiple Jira instances is offered for Mattermost Enterprise Edition E20.
```

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**